sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_store/feature_store.py

```diff
@@ -160,6 +160,7 @@ def __init__(
         self.identity = identity
         self.public_network_access = public_network_access
         self.managed_network = managed_network
+        self.compute_runtime = compute_runtime
```

Comment:
There is feature_store_settings property is supposed to set the spark runtime.

Reply:
here, compute_runtime is used instead of feature_store_settings because it uses default spark version if no compute_runtime is specified during update